.travis.yml

```diff
@@ -1,6 +1,6 @@
 # This config file for Travis CI utilizes https://github.com/ros-planning/moveit_ci/ package.
 sudo: required
-dist: trusty
+dist: xenial
```

Comment:
Were the tests running on trusty before this PR? Or am i misunderstanding what the `dist` option does?

Reply:
This simply refers to the distro ran by Travis. As we run our tests in docker, this is not really critical.
But, Travis folks indicated that they have some unresolved issues with their trusty image.